rmgpy/molecule/molecule.pxd

```diff
@@ -79,6 +79,8 @@ cdef class Atom(Vertex):
     cpdef bint is_nos(self)
 
     cpdef bint is_surface_site(self)
+
+    cpdef bind is_bonded_to_surface(self)
```

Comment:
This PR is consistent with the rest of the file so I'm not requesting you change it before we merge, but do you have any idea why `molecule.pxd` doesn't have the `except -2`  declarations but `group.pxd` does?

Reply:
```suggestion
    cpdef bint is_bonded_to_surface(self)
```

Reply:
Thanks, I just fixed the type with force push, and nope, I am not sure why that is